app/renderer/js/main.js

```diff
@@ -488,9 +489,14 @@ class ServerManagerView {
 
 window.onload = () => {
 	const serverManagerView = new ServerManagerView();
+	const reconnectUtil = new ReconnectUtil(serverManagerView);
 	serverManagerView.init();
 	window.addEventListener('online', () => {
-		serverManagerView.reloadView();
+		reconnectUtil.pollInternetAndReload();
```

Comment:
Is there a need for adding a console log, just to make the debugging better (for issues like https://github.com/zulip/zulip-electron/issues/312) -
`console.log('You're back online.')`
`console.log('No internet connection, you are offline.')`

Reply:
Ah yeah, so its easier to figure out.